apps/condo/domains/miniapp/schema/B2CApp.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Generated by `createschema miniapp.B2CApp 'name:Text;'`
+ */
+
+const dayjs = require('dayjs')
+const { Text, Relationship } = require('@keystonejs/fields')
+const { GQLListSchema, find, getByCondition } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+const access = require('@condo/domains/miniapp/access/B2CApp')
+const {
+    LOGO_FIELD,
+    SHORT_DESCRIPTION_FIELD,
+    DEVELOPER_FIELD,
+    IS_HIDDEN_FIELD,
+} = require('@condo/domains/miniapp/schema/fields/integration')
+const { COLOR_SCHEMA_FIELD } = require('@condo/domains/miniapp/schema/fields/b2cApp')
+const { B2CAppBuild, B2CAppProperty } = require('@condo/domains/miniapp/utils/serverSchema')
+const { RESTRICT_BUILD_APP_CHANGE_ERROR, RESTRICT_PROPERTY_APP_CHANGE_ERROR, RESTRICT_BUILD_SELECT_ERROR } = require('@condo/domains/miniapp/constants')
+
+
+const B2CApp = new GQLListSchema('B2CApp', {
+    schemaDoc: 'B2C App',
+    fields: {
+        name: {
+            schemaDoc: 'Name of B2C App',
+            type: Text,
+            isRequired: true,
+        },
+        logo: {
+            ...LOGO_FIELD,
+            isRequired: true,
+        },
+        shortDescription: SHORT_DESCRIPTION_FIELD,
+        developer: DEVELOPER_FIELD,
+        isHidden: IS_HIDDEN_FIELD,
+        colorSchema: COLOR_SCHEMA_FIELD,
+        currentBuild: {
+            schemaDoc: 'Link to current active app build',
+            type: Relationship,
+            ref: 'B2CAppBuild',
+            isRequired: false,
+            kmigratorOptions: { null: true, on_delete: 'models.PROTECT' },
+            hooks: {
+                validateInput: async ({ resolvedData, fieldPath, addFieldValidationError, operation, existingItem }) => {
+                    const appId = operation === 'create' ? resolvedData.id : existingItem.id
+                    if (resolvedData.hasOwnProperty(fieldPath) && resolvedData[fieldPath] !== null) {
+                        const validBuild = await getByCondition('B2CAppBuild', {
+                            deletedAt: null,
+                            app: { id: appId },
+                            id: resolvedData[fieldPath],
+                        })
+                        if (!validBuild) {
+                            return addFieldValidationError(RESTRICT_BUILD_SELECT_ERROR)
+                        }
+                    }
+                },
+            },
+        },
+        builds: {
```

Comment:
Please remove this field. It's too heavy.

Reply:
removed